CONTRIBUTING.md

```diff
@@ -76,7 +76,9 @@ Follow these steps to set up your local copy of the repository:
 
 1. [Fork this repository](https://docs.github.com/en/get-started/quickstart/fork-a-repo) and clone your fork.
 
-1. Navigate to your cloned repository.
+2. Navigate to your cloned repository.
```

Comment:
Please leave the 1. numbering. Jekyll automatically translates these into numbered lists. And it's easier to reorder/add/delete items this way.

Reply:
Reverted :D